docs/devel/api-conventions.md

```diff
@@ -423,6 +442,9 @@ APIs may return alternative representations of any resource in response to an Ac
 
 All dates should be serialized as RFC3339 strings.
 
+## Time Field Names
+
+The name of a field that specifies the time at which `something` occurs should be called `somethingTime`. Do not use `stamp` (e.g., `creationTimestamp`).
```

Comment:
We use at least 3 different conventions in the API currently. I picked the most aesthetically pleasing one for new fields. Whether it's worth changing existing fields is another issue.